README.md

```diff
@@ -59,4 +59,12 @@
 - Use paren bracket style delimiters for preg_* functions and the like `preg_match('(^\d+)', $text)`
 - No trailing whitespace on any line
 - Every file must end with a single new line character
+
+- All PHP functions should have a return typehint if it returns a value.
+ 1. If a return typehint causes the line to exceed the maximum line length then the colon indicating a return type hint should stay with the closing parenthesis.
+ 1. If a function's arguments fit on a single line, the typehint should be on the next line and indented.
+ 1. If a function's arguments do not fit on a single line, the typehint should follow the closing parenthesis.
+ 1. In both cases 2 & 3, the opening brace should stay with the typehint.
```

Comment:
is there a case where it doesn't?

Reply:
No. just making it explicit what is expected.